docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM amd64/ubuntu:22.04
+
+RUN apt-get update && \
+    apt-get install -y wget && \
+    wget https://s3.us-west-2.amazonaws.com/downloads.yugabyte.com/repos/reporpms/yb-apt-repo_1.0.0_all.deb && \
+    apt-get install -y ./yb-apt-repo_1.0.0_all.deb && \ 
+    apt-get clean && \
```

Comment:
Can we get rid of this `apt-get clean` instance? We already have another `apt-get clean` towards the end.